### DISCUSSION TITLE
HighEntropy Blink IDL Extended Attributes

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#1544] [General]
I've found this https://chromium.googlesource.com/chromium/src/+/HEAD/third_party/blink/renderer/bindings/IDLExtendedAttributes.md#highentropy_m_a_c

**Summary: Denotes an API that exposes data that folks on the internet find useful for fingerprinting.**

In blink, data that could be used for fingerprinting via javascript are marked
https://source.chromium.org/search?q=HighEntropy%3DDirect&ss=chromium%2Fchromium%2Fsrc

**Comment by csagan5** [comment]
Yes, this is part of an effort to give each website a "budget" of high-entropy information to detect fingerprinting.
